src/commands/mod.rs

```diff
@@ -171,6 +173,19 @@ pub enum InsaneFeeInfo {
     TooHighFeerate(u64),
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum RbfErrorInfo {
+    TooLowFeerate(u64),
```

Comment:
I think it would be helpful to include the min feerate here as well as / instead of the chosen feerate.

Reply:
I haven't changed this yet. We could leave for a follow-up.